index.js

```diff
@@ -439,6 +447,12 @@ Converter.prototype.buildParameter = function(oldParameter) {
       'Complex type is used inside non-body argument.');
   }
 
+  //According to Swagger 2.0 spec: If the parameter is in "path",
+  //this property is required and its value MUST be true.
+  if (parameter.in === 'path') {
```

Comment:
@webron the `required` property is not required. This is valid in editor at least:


```yaml
---
swagger: '2.0'
info:
  version: 0.0.0
  title: Simple API
paths:
  /{foo}:
    parameters:
      - name: foo
        in: path
        type: string
      
    get:
      responses:
        200:
          description: OK
```